src/app/cube/content-pages/editorial-process/copy.ts

```diff
@@ -0,0 +1,154 @@
+export const sections = {
```

Comment:
Can we edit this page for spacing consistency?

Reply:
This is fine. This is the format of github, not the code